### PR TITLE
Component/link

### PR DESCRIPTION
**Pull request by HQCuong**

### CODE REVIEW

File: src/components/Link/type.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
+/**
```
Comment (juzser) [inline src/components/Link/type.ts:1]: Filename should be `types.ts`

**Review by juzser — CHANGES_REQUESTED** [review]

File: docs/components/Link/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+</Examples>
+
+## Props
+
+<PropsTable />
+
+## Slots
+
+<SlotsTable />
+
```
Comment (haild173094) [inline docs/components/Link/README.md:41]: Check more info in link.mdx file

File: src/components/Link/Link.vue [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+  styles.Link,
+  shouldBeMonochrome.value && styles.monochrome,
+  props.removeUnderline && styles.removeUnderline,
+));
+</script>
+
+<style lang="scss" module>
+@import '@polaris/components/Link/Link.scss';
+</style>
+./types
```
Comment (juzser) [inline src/components/Link/Link.vue:95]: .

File: src/components/Link/types.ts [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+interface RouteQueryAndHash {
+  query?: LocationQueryRaw;
+  hash?: string;
+}
+
+/**
+ * Common options for all navigation methods.
+ */
+export declare interface RouteLocationOptions {
+    /**
```
Comment (juzser) [inline src/components/Link/types.ts:44]: tab = 4 spaces?

**Review by juzser — CHANGES_REQUESTED** [review]

File: docs/components/Link/README.md [hunk 5]
```diff
@@ -0,0 +1,194 @@
...
+## Best practices
```
Comment (juzser) [inline docs/components/Link/README.md:42]: Wrap additional contents inside `<div font-size...` element to reduce font size

**Review by juzser — CHANGES_REQUESTED** [review]

**Review by haild173094 — APPROVED** [review]

**Review by juzser — APPROVED** [review]

**Pull request merged**